en/docs/guides/user-mgt/enable-account.md

```diff
@@ -0,0 +1,123 @@
+# Enable and Disable User Accounts
+
+Account locking and account disabling are security features in WSO2 Identity Server (IS) that can be used to prevent users from logging in to their account and from authenticating themselves using their WSO2 IS account. The account locking feature is used to **temporarily** block a user from logging in, for example, in instances where there have been many consecutive, unsuccessful login attempts.
+
+## Disable user accounts using the admin portal
+
+{insert-fragment}
```

Comment:
shall we include the necessary fragments in the same PR so that the reference is valid and we can avoid errors ?

Reply:
The time out errors were due to giving file names that were non existent. These fragments are from the admin portal so we can't add them till those docs are ready.